internal/jobservice/application.go

```diff
@@ -55,10 +55,13 @@ func (a *App) StartUp(ctx context.Context, config *configuration.JobServiceConfi
 		return err
 	}
 
-	g.Go(func() error {
-		PurgeJobSets(ctx, log, config.PurgeJobSetTime, sqlJobRepo)
-		return nil
-	})
+	if config.DatabaseType != "postgres" {
+		g.Go(func() error {
+			PurgeJobSets(ctx, log, config.PurgeJobSetTime, sqlJobRepo)
```

Comment:
I don't think this works for sqlite at the moment.  Maybe we just remove `PurgeJobSets` and say that if you are using sqllite deletion will not happen automically.

Reply:
added trigger for sqlite too